subjects/bowling_score/README.md

```diff
@@ -0,0 +1,106 @@
+## bowling_score
+
+### Instructions
+
+Create a library that implements a `struct` to calculate the score of a player in the game of [Bowling](https://en.wikipedia.org/wiki/Bowling).
+
+The methods of `BowlingGame` will be:
+
+- `roll`: It will take the number of knocked pins and return `Ok(())` if the roll is valid or an `Err(Error::...)` if the roll is invalid.
+- If the pins knocked out are more than the available ones the error will be `NotEnoughPinsLeft`.
```

Comment:
"If more pins are knocked down than are available, the error will be..."

Reply:
I've rephrased it a bit. done.